### PR TITLE
feat: get transfer status function

### PR DESCRIPTION
**Pull request by wainola**
## Description

This PR implements utility function to get transfer status

## Related Issue Or Context

Closes: #307 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation

## Checklist:

- [ ] I have commented my code, particularly in hard-to-understand areas.
- [ ] I have ensured that all acceptance criteria (or expected behavior) from issue are met
- [ ] I have updated the documentation locally and in chainbridge-docs.
- [ ] I have added tests to cover my changes.
- [ ] I have ensured that all the checks are passing and green, I've signed the CLA bot

### CODE REVIEW

File: packages/sdk/src/utils.ts [hunk 1]
```diff
@@ -15,3 +16,22 @@ export function getFeeOracleBaseURL(environment?: Environment): string {
       return MAINNET_FEE_ORACLE_BASE_URL;
   }
 }
+
+export async function getTransferStatus(
```
Comment (MakMuftic) [inline packages/sdk/src/utils.ts:20]: I would add docs to this function, explaining that txHash is deposit transaction txHash

**Review by MakMuftic — COMMENTED** [review]
Looks good in general @wainola. One small nitpick.

**Comment by wainola** [comment]
@MakMuftic according to updates on the endpoint from indexer, `domainId` is going to be a query param, I guess we can add this parameter to the function as optional, lmk your thougths

**Review by MakMuftic — APPROVED** [review]

**Comment by MakMuftic** [comment]
@wainola just please fix lint error :pray:

**Comment by wainola** [comment]
> @MakMuftic @FSM1 there are some linter errors that I need to fix and they are appearing for some reason. Could be related to the update of the ethers version

Fixed

**Review by MakMuftic — DISMISSED** [review]

**Review by MakMuftic — DISMISSED** [review]

File: examples/evm-to-evm-fungible-transfer/src/transfer.ts [hunk 2]
```diff
@@ -14,12 +18,26 @@ const SEPOLIA_CHAIN_ID = 11155111;
...
+const getStatus = async (
+  txHash: string
+): Promise<{ status: string; explorerUrl: string } | void> => {
+  try {
+    const data = await getTransferStatusData(Environment.TESTNET, txHash);
+
+    return data as { status: string; explorerUrl: string };
+  } catch (e) {
+    console.log("error", e);
+    console.log("indexing and retrying");
```
Comment (FSM1) [inline examples/evm-to-evm-fungible-transfer/src/transfer.ts:30]: What exactly are we indexing here?

Comment (wainola) [reply examples/evm-to-evm-fungible-transfer/src/transfer.ts:30]: will rephrase it

**Review by FSM1 — APPROVED** [review]

**Review by MakMuftic — APPROVED** [review]

**Pull request merged**